recipes/capnproto/all/conanfile.py

```diff
@@ -141,12 +142,23 @@ def package(self):
                 os.remove(cmake_file)
         # inject mandatory variables so that CAPNP_GENERATE_CPP function can
         # work in a robust way (build from source or from pre build package)
+        find_execs = textwrap.dedent("""\
+            if(CMAKE_CROSSCOMPILING)
+                find_program(CAPNP_EXECUTABLE capnp PATHS ENV PATH NO_DEFAULT_PATH)
+                find_program(CAPNPC_CXX_EXECUTABLE capnpc-c++ PATHS ENV PATH NO_DEFAULT_PATH)
+            endif()
+            if(NOT CAPNP_EXECUTABLE)
+                set(CAPNP_EXECUTABLE "${CMAKE_CURRENT_LIST_DIR}/../../../bin/capnp${CMAKE_EXECUTABLE_SUFFIX}")
```

Comment:
is this relative path stable? what is the expected location of `CMAKE_CURRENT_LIST_DIR`?

Reply:
`<package_folder>/lib/cmake/CapnProto`

Reply:
This is shockingly stable because of the fixed package layout... I had the same reaction when I saw this in the Qt recipes